run/faasm.py

```diff
@@ -1,104 +0,0 @@
-#!/usr/bin/python3
-import json
-import math
-import re
-import sys
-import time
-
-from os import environ
-from os.path import dirname, join, realpath
-from subprocess import check_output
-
-from faasmcli.util.call import invoke_impl
-
-PROJ_ROOT = dirname(dirname(realpath(__file__)))
-BASE_DIR = "{}/../..".format(PROJ_ROOT)
-LAMMPS_USR = "lammps"
-LAMMPS_FUNC = "main"
-LAMMPS_CMDLINE = "-in faasm://lammps-data/in.controller"
-
-# Benchmark details
-# TODO decide where common config may live
-NUM_PROCS = [1, 2, 4]
-NUM_TESTS = 1
-
-
-# Update k8s host and port env variables
-def update_env():
-    _out = check_output(
-        "{}/faasm/bin/knative_route.sh | tail -6".format(BASE_DIR), shell=True
-    )
-    out = _out.decode("utf-8").strip().split("\n")
-
-    # Prepare dict
-    new_env = {}
-    for var in out:
-        new_env[var.split(" ")[0].upper()] = var.split(" ")[2]
-
-    # Update environment
-    environ.update(new_env)
-
-
-def invoke(np):
-    """
-    Invoke one LAMMPS execution
-    """
-    start = time.time()
-    cmd_out = invoke_impl(
-        LAMMPS_USR, LAMMPS_FUNC, mpi_world_size=np, cmdline=LAMMPS_CMDLINE, debug=True
-    )
-    print(cmd_out)
-    elapsed_mins = math.ceil((time.time() - start) / 60)
```

Comment:
I can't work out the benefit of using minutes here, this loses a lot of granularity. I've now switched to using seconds.